src/render.rs

```diff
@@ -53,8 +53,7 @@ impl<'a> MarkdownRenderer<'a> {
             .cloned()
             .collect();
         let tag_attributes = [
-            ("a", ["href", "target"].iter().cloned().collect()),
-            ("code", ["class"].iter().cloned().collect()),
```

Comment:
hm, does `allowed_classes` make this line unnecessary...? I guess so since the tests pass!

Reply:
Indeed, it even became required in the newer `ammonia`: https://github.com/notriddle/ammonia/blob/2581eec04ef98419fd202e0dd4d8997c9dfdb06e/src/lib.rs#L1030-L1037 (you're allowed to use `allowed_classes` or set `class` as an allowed tag, but not both)